src/abc_inference.jl

```diff
@@ -0,0 +1,23 @@
+function createabcfunction(prob, t, distancefunction, alg, kwargs...)
+    function simfunc(params, constants, targetdata)
+        sol = solve(problem_new_parameters(prob, params), alg, saveat = t, kwargs...)
```

Comment:
do `save_start = false`. Then you don't have to slice `[:, 2:end]` in the next line, and you'll save a matrix allocation.